dist/docker/Makefile

```diff
@@ -104,13 +105,17 @@ ubuntu-20.04: Dockerfile.focal
 	@echo ===================================================================================
 	${DOCKER} build -f $< ${DOCKER_ARGS} -t ${DOCKER_IMAGE}-$@ ${DOCKER_CONTEXT}
 
+ubuntu-22.04: Dockerfile.jammy
+	@echo ===================================================================================
+	${DOCKER} build -f $< ${DOCKER_ARGS} -t ${DOCKER_IMAGE}-$@ ${DOCKER_CONTEXT}
+
 archlinux: Dockerfile.archlinux
 	@echo ===================================================================================
 	${DOCKER} build -f $< ${DOCKER_ARGS} -t ${DOCKER_IMAGE}-$@ ${DOCKER_CONTEXT}
 
 release:
 	@test -f setup.sh || echo 'Error: require setup.sh for user authentication'
-	for dist in ubuntu-16.04 ubuntu-18.04 ubuntu-20.04; do \
+	for dist in ubuntu-16.04 ubuntu-18.04 ubuntu-20.04 ubuntu-22.04; do \
```

Comment:
If the `ubuntu-22.04` build still has some issues (as you mention in the commit text), does it even make sense to add it to the list here?

Reply:
I was waiting on #92, then I'll rebase and make server optional in jammy.